app/code/Magento/Catalog/Block/Adminhtml/Product/Edit/Tab/Options/Option.php

```diff
@@ -14,6 +14,8 @@
 use Magento\Catalog\Api\Data\ProductCustomOptionInterface;
 
 /**
+ * Class \Magento\Catalog\Block\Adminhtml\Product\Edit\Tab\Options\Option
```

Comment:
We don't need such obvious info in doc block. There should be an informative description of what this class for